### DISCUSSION TITLE
Loco position anchor can't be detected

### DISCUSSION THREAD

**Discussion by Xinlyu123** [bitcraze/discussions#165] [Q&A]
Hi guys! I need your help.
I browsed the bitcraze forum I didn't find similar case.
The issue I am facing right now is the loco position anchors can't be detected. It works fine with crazyfile that we had in the lab. However, we made a drone with crazyfile bolt 1.1 and the loco position deck can't be detected. I tried flash all the anchors and it was working(All 8 anchors can be detected). However, the next day I came in and some of the deck can't be detected.
We have 8 loco position anchors and only 4 of them can be detected right now.
What are some possible reasons that cause this?
The power source is fine.

**Comment by Xinlyu123** [comment]
Sorry for the confusion I mean loco position anchor not loco position deck

**Reply by knmcguire** [reply]
I've changed it in the original text so it is more clear :)

**Comment by knmcguire** [comment]
Hi!

That is indeed not handy. Here some questions to see what is going on:
- Do you see an difference with the LED sequence with the anchors that are detected and the ones that aren't?
- After flashing them, have you changed the mode within the [cfclient loco positioning tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/loco_tab/)?
- Have you tried any of the things listed in the troubleshooting guide here: https://www.bitcraze.io/documentation/repository/lps-node-firmware/master/user-guides/configure-mode/ ?
